src/protocols/coda_pow.ml

```diff
@@ -1217,7 +1217,9 @@ module type Consensus_mechanism_intf = sig
     -> local_state:Local_state.t
     -> keypair:keypair
     -> logger:Logger.t
-    -> [`Check_again of Int64.t | `Propose of Int64.t * Proposal_data.t]
+    -> [ `Check_again of Int64.t
+       | `Propose_now of Int64.t * Proposal_data.t
+       | `Propose of Int64.t * Proposal_data.t ]
```

Comment:
We should probably refer to `Unix_timestamp.t` here. It's equivalent, but the module name was added to be specific about what the `Int64.t` represented.